src/app/tables/tests-table/tests-table.component.ts

```diff
@@ -65,6 +65,8 @@ export class TestsTableComponent implements OnChanges {
           'Test Case': result.testcase.name,
           'Logs': 'No logs available',
           'Waived': waivedTestCases.includes(result.testcase.name) ? 'Yes' : 'No',
+          'Impacts the decision': this.greenwaveDecision.satisfied_requirements.map(
+            function (j) { return j.result_id; }).includes(result.id) ? 'Yes' : 'No',
```

Comment:
Could the map function be run once outside of the for loop and stored in a variable to avoid performing it for ever test result?